framework/src/modules/chain/processor/processor.js

```diff
@@ -184,7 +184,7 @@ class Processor {
 	}
 
 	// processValidated processes a block assuming that statically it's valid
-	async processValidated(block) {
+	async processValidated(block, removeFromTempTable = false) {
```

Comment:
Can we include this property in an object literal? This way we can name the parameters.

Reply:
Yes will do, thanks!